hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServletInodeBasedXfer.java

```diff
@@ -297,6 +297,16 @@ private void transferSnapshotData(Set<String> sstFilesToExclude, Path tmpdir, Se
         om.getOmSnapshotManager().invalidateCacheEntry(UUID.fromString(snapshotId));
         writeDBToArchive(sstFilesToExclude, snapshotDir, maxTotalSstSize, archiveOutputStream, tmpdir,
             hardLinkFileMap, false);
+        Path snapshotDirParent = snapshotDir.getParent();
+        if (snapshotDirParent != null) {
+          Path snapshotLocalPropertyYaml = snapshotDirParent.resolve(
+              snapshotDir.getFileName() + ".yaml");
```

Comment:
Could you reuse `OmSnapshotManager.getSnapshotLocalPropertyYamlPath()` here? That way, any potential changes to the yaml file path will be reflected here without modifying the path in multiple places.

Reply:
LGTM.